tasks/lint_rules/src/generate_list/typescript.rs

```diff
@@ -0,0 +1,67 @@
+use oxc_allocator::Allocator;
+use oxc_ast::{ast::ObjectPropertyKind, syntax_directed_operations::PropName, AstKind};
+use oxc_parser::Parser;
+use oxc_semantic::SemanticBuilder;
+use oxc_span::SourceType;
+use std::collections::HashSet;
+
+const ORIGINAL_JS_SOURCE_URL: &str =
```

Comment:
I wonder if there is another venue for getting the "all" list marking which ones are recommended. Otherwise we'll need to parse two files 😅

Reply:
> need to parse two files

How about using JS runtime(instead of Rust + AST parsing)?
Actually clone plugin repo and `import`(or `require`) all rule files and check its `meta.recommended` prop.

At first I thought this was a wild idea, but then I found out that `jest` does not provide a static rule list, so maybe...?

> https://github.com/jest-community/eslint-plugin-jest/blob/main/src/index.ts

(Here, as with `eslint`, does this mean support for `all` as well as `recommended`? 👀 )